ci/OWNERS

```diff
@@ -27,7 +27,7 @@
 
 # Libraries
 /lib                        @infinisil
-/lib/systems                @alyssais @ericson2314
+/lib/systems                @alyssais @ericson2314 @NixOS/stdenv
```

Comment:
CC @alyssais @Ericson2314 on this one. Is it redundant to have a member of the stdenv team mentioned while also mentioning the stdenv team itself?

Reply:
It is redundant, unless of course the member leaves the team,